requirements-test.txt

```diff
@@ -8,4 +8,4 @@ awscli==1.33.33
 nbval==0.11.0
 # For NvFaidx equivalence tests
 pyfaidx==0.8.1.3
-wandb<0.19.0 # temporary pin: https://nvidia.slack.com/archives/C074Z808N05/p1733418209959769
```

Comment:
Actually, could you delete the line here and move the new pin to `requirements-cve.txt`?

Reply:
Done'd.